pic_database.php

```diff
@@ -0,0 +1,61 @@
+<?php
+	include('config/config.php');
+	include('include/db_query.php');
+
+	// InsertPic('Wes Hicks', '2018-05-25', 'TestPic', 'pictures/0f191f-wes-hicks-464614-unsplash.jpg', 'flavortext');
+	$pic = GetPic($_REQUEST['picID']);
+	//var_Dump($pic);
+	DisplayPic($pic);
+	// echo"<img src='pictures/0f191f-wes-hicks-464614-unsplash.jpg'alt='".$pic['flavor']."'>";
+
+
+	function InsertPic($photographer, $date, $title, $link, $flavor){
+			$result = dbQuery("
+				INSERT INTO pics (photographer, date, title, link, flavor)
+				VALUES('$photographer', '$date', '$title', '$link', '$flavor')
+			")->fetch();
+		}
+
+		function GetPic($picID){
```

Comment:
This is super minor and just a person opinion, but I'd argue that "ID" should be written "Id". There'a often a question of whether or not to capitalize each letter in an acronym. My argument for only capitalizing the first letter is that it makes it easier to read other words in the variable name. For example, let's say an API returns a JSON result. You could name the variable `$APIJSON` or `$ApiJson`. I find the latter easier to read. Totally your call though, there's not a right or wrong approach.